steps/deletedriversisoonesxihostwithmacosorlinuxworker/script.txt

```diff
@@ -1,7 +1,7 @@
-cd "{automationWorkerLinuxBaseDirectory}"
+cd "{automationWorkerWindowsBaseDirectory}"
 if [ ! -f drivers_{newVmNode.fqn}.iso ]
 then
-    echo "This build doesn't have a {automationWorkerLinuxBaseDirectory}/drivers_{newVmNode.fqn}.iso."
+    echo "This build doesn't have a {automationWorkerWindowsBaseDirectory}/drivers_{newVmNode.fqn}.iso."

```

Comment:
Replace the `{automationWorkerWindowsBaseDirectory}` with `C:\attuneautomationworker`

Reply:
Oh no. Is that the directory name?

Why not C:\attune_automation_worker ?

Reply:
@brentonford and @jarrodchesney I just realized it's my error. The step is for macOS or Linux Worker so it should be using `{automationWorkerLinuxBaseDirectory}`.

Reply:
Hi @brentonford here is the commit `36072b1c0fd99d7d231adaebfc627bfa08ec0ad9` with the correction to my errors:
![image](https://github.com/Attune-Automation/VMWare-ESXi-APIs/assets/14309370/734ebd4a-9da4-41ff-b0ad-ba68c2b7f57d)